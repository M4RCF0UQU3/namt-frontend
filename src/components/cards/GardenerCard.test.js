import React from 'react';
import Card, { CardActions, CardContent, CardMedia } from 'material-ui/Card';
import MainPageCard from './MainPageCard.jsx';
import Typography from 'material-ui/Typography';
import Button from 'material-ui/Button';
import renderer from 'react-test-renderer';

it('GardenerCard correctly', () => {
  const tree = renderer
    .create(<MainPageCard>
		        <CardMedia
		          image="/images/aqueuse.jpg"
		          title="Blog"
		        />
		      <CardContent>
		          <Typography type="headline" component="h3">
		            Boule d'eau
		          </Typography>
		          <Typography component="p">
		            Chercher un jardin.
		          </Typography>
		        </CardContent>
		        <CardActions>
		          <div />
		          <Button label="Remplir le formulaire"
		              href="https://docs.google.com/forms/d/e/1FAIpQLScARUhLXGZmND_pBvti02oi_sZvJce35jnfejglIAY4FHxxSQ/viewform"
		                target="_blank">Je cherche jardin (Ãªtre inscrit)</Button>
		        </CardActions>
			</MainPageCard>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});