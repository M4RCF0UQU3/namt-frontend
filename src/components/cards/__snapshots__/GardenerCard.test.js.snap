// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GardenerCard correctly 1`] = `
<div
  className="MuiPaper-root-6 MuiPaper-shadow2-10 MuiPaper-rounded-7"
  style={
    Object {
      "margin": "auto 10px",
      "marginBottom": "20px",
      "minHeight": "400px",
    }
  }
>
  <div
    className="MuiCardMedia-root-33"
    src={undefined}
    style={
      Object {
        "backgroundImage": "url(/images/aqueuse.jpg)",
      }
    }
    title="Blog"
  />
  <div
    className="MuiCardContent-root-35"
  >
    <h3
      className="MuiTypography-root-36 MuiTypography-headline-41"
    >
      Boule d'eau
    </h3>
    <p
      className="MuiTypography-root-36 MuiTypography-body1-45"
    >
      Chercher un jardin.
    </p>
  </div>
  <div
    className="MuiCardActions-root-60"
  >
    <div
      className="MuiCardActions-action-61"
    />
    <a
      className="MuiButtonBase-root-78 MuiButton-root-62 MuiCardActions-action-61"
      href="https://docs.google.com/forms/d/e/1FAIpQLScARUhLXGZmND_pBvti02oi_sZvJce35jnfejglIAY4FHxxSQ/viewform"
      label="Remplir le formulaire"
      onBlur={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      target="_blank"
    >
      <span
        className="MuiButton-label-64"
      >
        Je cherche jardin (Ãªtre inscrit)
      </span>
      <span
        className="MuiTouchRipple-root-80"
      />
    </a>
  </div>
</div>
`;
